bears/c_languages/CSecurityBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable="flawfinder", output_format="regex",
+        output_regex=r'.+:(?P<line>\d+):(?P<column>\d+):\s*'
+                     r'\[(?P<severity>\d)\]\s*'
+                     r'\((?P<origin>.+)\) (?P<message>.+)',
+        severity_map={"5": RESULT_SEVERITY.MAJOR,
+                      "4": RESULT_SEVERITY.NORMAL, "3": RESULT_SEVERITY.NORMAL,
+                      "2": RESULT_SEVERITY.INFO, "1": RESULT_SEVERITY.INFO})
+class CSecurityBear:
+    """
+    Reports possible security weaknesses for C like languages.
```

Comment:
I wouldnt call it C like ... C like would include C#, Obj-C, etc right ?
This is for C and C++ only according to the documentation page